src/runtime/pkg/direct-volume/utils.go

```diff
@@ -17,6 +17,19 @@ import (
 
 const (
 	mountInfoFileName = "mountInfo.json"
+
+	FSGroupMetadataKey             = "fsGroup"
+	FSGroupChangePolicyMetadataKey = "fsGroupChangePolicy"
+)
+
+type FSGroupChangePolicy string
```

Comment:
Should there be a comment here indicating that `FSGroupChangePolicy` and its values below roughly tracks https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/api/core/v1/types.go#L3244 and it is up to the CSI plugin to either directly map to the corresponding fields in a pod or specify the most optimal setting?

Reply:
Thanks, added comment indicating `FSGroupChangePolicy` and values track the `PodFSGroupChangePolicy` defined in:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/types.go